umbraco/Umbraco-CMS
# Show tracked references on a content item

### Prerequisites


If there's an existing issue for this PR then this fixes #8692 

### Description

This is the first PR in a series of PR's related to this bigger task #8679 

Starting out I noticed that all the code related to showing references on a media item was strongly coupled to the media section.

To decouple this I first created a dedicated api controller and angular resource for getting references.
After that I created a reusable angular component to show the references.

I refactored the info tab for media to decouple showing of the references by using this component. After that I also used the component on the info tab of content items.

Now references to the content item are visible on the info tab.

![image](https://user-images.githubusercontent.com/1193822/90893222-e4caaa80-e3be-11ea-8f4d-331e22677309.png)

### Testing

- Check if media item references are shown correctly on the info tab
- Check if content item references are shown correctly on the info tab

Both can be verified by checking the relations "Related Document" and "Related Media" 

Probably best to test with multi variant site as well
